source/includes/v2/_sources.md

```diff
@@ -78,16 +79,16 @@ The monitoring API gives access to the data of one specific dataset containing t
 
 ## Opendatasoft source
 
-> Get a list of all public datasets on Opendatasoft Data Hub
+> Get a list of all public datasets on the Opendatasoft Data Network
```

Comment:
I'm used to use "Data network" but if you've already check you can ignore :)

Reply:
True :)
```suggestion
> Get a list of all public datasets on Opendatasoft's Data Network
```